PSL/SENSORS/MPU6050.py

```diff
@@ -33,8 +33,8 @@ def __init__(self, I2C, **args):
         self.I2C = I2C
         self.ADDRESS = args.get('address', self.ADDRESS)
         self.name = 'Accel/gyro'
-        self.params = {'powerUp': ['Go'], 'setGyroRange': [250, 500, 1000, 2000], 'setAccelRange': [2, 4, 8, 16],
-                       'KalmanFilter': [.01, .1, 1, 10, 100, 1000, 10000, 'OFF']}
+        self.params = {'powerUp': None, 'setGyroRange': [250, 500, 1000, 2000], 'setAccelRange' : [2, 4, 8, 16],
+                       'KalmanFilter': 'double' }
```

Comment:
In pslab-desktop-app  how would GUI know the range of slider that it needs to generate in KalmanFilter value? ( if a text box is generated, to take input from user then also we need to insert a check that user doesn't give some invalid value )